src/trans/codegen_c.cpp

```diff
@@ -1993,29 +1993,35 @@ namespace {
             m_mir_res = &top_mir_res;
             TRACE_FUNCTION_F(p);
 
-            if( item.m_linkage.name != "" && m_compiler != Compiler::Gcc )
+            // LLVM supports prepending a symbol name with \1 to prevent further mangling.
+            // Since we're targeting C, not LLVM, strip off this prefix.
+            std::string linkage_name = item.m_linkage.name;
+            if( !linkage_name.empty() && linkage_name[0] == '\1' ) {
+                linkage_name = linkage_name.substr(1);
+            }
+
+            if( linkage_name != "" && m_compiler != Compiler::Gcc )
             {
                 switch(m_compiler)
                 {
                 case Compiler::Gcc:
                     // Handled with asm() later
                     break;
                 case Compiler::Msvc:
```

Comment:
This is now unreachable due to the above if.

Reply:
The `if` was already there -- I changed `item.m_linkage.name` to `linkage_name` in the condition.

But you're right, it was unreachable, so I'll remove the check for Gcc in the `if`.